readme.md

```diff
@@ -37,6 +38,29 @@ This project is a Discord bot built with `discord.js`. The bot includes various
    - **DISCORD_CLIENT_ID**: Your bot's client ID.
    - **DISCORD_GUILD_ID**: The ID of the server (guild) where the commands will be deployed.
 
+To obtain each of these environment variables, you need to have `Developer Mode` enabled in Discord (if it is not already).
+
+Open Discord to enable developer mode and click on the gear icon (User Settings) at the bottom left. Go to `App Settings` and select the `Advanced` option, then look for the `Developer Mode` option and toggle it on. For more details on how to create a Discord bot, you can visit the official Discord guide.
+
+Once `Developer Mode` is enabled, you can follow these steps to obtain Environment Variables:
+
+### 1. Obtain **DISCORD_TOKEN**
+- Go to the [Discord Developer Portal](https://discord.com/developers/applications) and log in.
+- Create a new application by clicking on “New Application”.
+- Name your application and click "Create".
+- In the "Bot" section, click on "Add Bot" and confirm.
```

Comment:
I suggest a link to an official tutorial as complement, or leave images as references of where to find the bet section visually.